znoyder/config.yml

```diff
@@ -386,6 +386,18 @@ override:
             PYTHONPATH: /usr/share/openstack-dashboard
             TOX_TESTENV_PASSENV: PYTHONPATH
 
+  'networking-l2gw':
+    'osp-17.0':
+      'osp-rpm-py39':
+        vars:
+          extra_commands:
+            - dnf install -y python3-neutron-tests python3-neutron-lib-tests python3-neutronclient-tests vim
```

Comment:
I'm all for vim, but I'm not sure we need it in the CI jobs :P

Reply:
Without that, the unittests were failing on subprocess call `which vim` :D